plenum/server/consensus/ordering_service.py

```diff
@@ -1161,7 +1161,9 @@ def _apply_pre_prepare(self, pre_prepare: PrePrepare):
                                                                                            to_str=False),
                                                            primaries=[],
                                                            valid_digests=self._get_valid_req_ids_from_all_requests(
-                                                               reqs, invalid_indices))
+                                                               reqs, invalid_indices),
+                                                           digest=self.replica_batch_digest(reqs)
```

Comment:
I think `pre_prepare.digest` can be used instead

Reply:
ok